src/azure-cli/azure/cli/command_modules/network/aaz/latest/network/virtual_appliance/_reimage.py

```diff
@@ -46,7 +46,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
 
         _args_schema = cls._args_schema
         _args_schema.network_virtual_appliance_name = AAZStrArg(
-            options=["-n", "--name", "--network-virtual-appliance-name"],
+            options=["--network-virtual-appliance-name"],
```

Comment:
it seems we modified the `reimage` command, it's unrelated to the feature request.